doc/m4/FS22-Group-36-M4-Report.md

```diff
@@ -0,0 +1,97 @@
+\centering
+
+# SCREW YOUR NEIGHBOR
+
+Report for Milestone 3 of Sopra FS22, Group 36
+
+![cover (source B. Furrer)](../header/jass-image.png)
+
+**Members**
+
+Carmen Kirchdorfer (20-720-132)  
+Salome Wildermuth (10-289-544)  
+Beat Furrer, group leader (07-542-392)  
+Lucius Bachmann (11-060-274)  
+Moris Camporesi (19-764-349)
+
+\newpage
+\raggedright
+## 1.  Introduction to the game
+Our game can be played by two up to five people. The goal for every player is to gain as 
+many points as possible during the game. Every player plays for himself, 
+there are no teams. The game is played with a “Swiss Jass” card set of 36 cards with four suits.
+
+The game is divided into 9 matches (Runden) with changing number of distributed cards per player 
```

Comment:
```suggestion
The game is divided into 9 matches with changing number of distributed cards per player 
```

Reply:
We use this term in the meantime mainly for the round per trick (not for the match anymore).